packages/block-editor/src/layouts/flow.js

```diff
@@ -143,6 +151,8 @@ export default {
 					}
 					${ appendSelectors( selector, '> .alignfull' ) } {
 						max-width: none;
+						margin-left: calc( -1 * ${ padding?.left || 0 } ) !important;
+						margin-right: calc( -1 * ${ padding?.right || 0 } ) !important;
```

Comment:
The zero here is causing an invalid property value error.

Reply:
Switched this to [only add these margins when a padding is actually set](https://github.com/WordPress/gutenberg/pull/39871/commits/3304a3307d6b4d4d7fae917afe730fc1f85eae7d), but not sure if there is a need to have them actually set to 0 if no padding set? Also, in my testing the `padding` object always seems to be undefined regardless of what padding is set on parents.